config/development.env

```diff
@@ -0,0 +1,13 @@
+NODE_ENV=production
+OAUTH_CLIENT_ID=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHMwcQYJKoZIhvcNAQcGoGQwYgIBADBdBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDGM1ccp/vwImDcIFwQIBEIAwl1AV5yfZ/tquR4OY9w48ImVdZNi3wXluoY6LqyMDB5s1scHksWy1dmtZLjXAhRi8
+OAUTH_PROVIDER_URL=https://isso.nypl.org/oauth/token
+NYPL_DATA_API_URL=https://platform.nypl.org/api/v0.1/
```

Comment:
Are we sure NYPL_DATA_API_URL is just for getting info?

Reply:
I believe this service does look ups for patron and item record information to pass along to ReCAP's API. For development environments, the URL should be https://dev-platform.nypl.org/api/v0.1.